x-pack/plugin/security/src/main/plugin-metadata/entitlement-policy.yaml

```diff
@@ -4,6 +4,37 @@ org.elasticsearch.security:
       - relative_path: ""
```

Comment:
This should be deleted, or do we want to _also_ allow read access to all of the directory

Reply:
I suspect there are other files. Given that plugins currently have full access to the config dir, I would like to leave this in here for now, and followup with reducing it to just the files actually needed.